tests/containers/host_configuration.pm

```diff
@@ -25,9 +25,7 @@ sub run {
         ensure_ca_certificates_suse_installed();
         disable_and_stop_service(opensusebasetest::firewall, ignore_failure => 1);
     }
-    else {
-        # Re-running dhclient on RHEL is confusing the routing tables
-        assert_script_run "dhclient -v" unless get_var("HDD_1") =~ /rhel/;
+    elsif (get_var("HDD_1") =~ /rhel/) {
```

Comment:
For some reason, this `rhel` check is invalid and maybe it's some leftovers, so you can remove it.
What  you need to do is actually use `else` here.

```suggestion
    else {
```

Reply:
What I mean is, the rest of the commands are also needed for Leap/CentOS hosts...

Reply:
I need to exclude `ubuntu`:
```suggestion
    elsif (get_var("HDD_1") !~ /ubuntu/) {
```